components/engine_panic/src/db_options.rs

```diff
@@ -36,6 +36,14 @@ impl DBOptions for PanicDBOptions {
         panic!()
     }
 
+    fn get_auto_tuned(&self) -> Option<bool> {
```

Comment:
can we get a better name, like `get_rate_limiter_is_auto_tuned`?

Reply:
It does look a little off, I'll try changing the config name entirely to avoid similar confusion.

Reply:
Change made, I'm now sure how we handle the backward compatibility in this case, what do you think? @yiwu-arbug

Reply:
Since the underlying approach is changed, I think it is fine to not providing the old config name.

Reply:
regardless of whether the old auto-tuned was enabled, since we rename the config, all users will be using the new auto-tuned by default, which I don't see the down side.